packages/components/package.json

```diff
@@ -0,0 +1,52 @@
+{
+    "name": "components",
+    "private": true,
+    "version": "0.0.0",
+    "type": "module",
+    "files": [
+        "dist",
+        "tailwind.config.js"
+    ],
+    "main": "./dist/component-library.cjs",
+    "module": "./dist/component-library.js",
+    "exports": {
+        "./content": {
+            "import": "./.contentlayer/generated/index.mjs",
+            "types": "./.contentlayer/generated/index.d.ts"
+        },
+        ".": {
+            "import": "./dist/lib/component-lib.js",
+            "types": "./dist/lib/component-lib.d.ts"
+        },
+        "./styles": "./dist/lib/index.css",
+        "./tailwind": "./tailwind.config.js"
+    },
+    "scripts": {
+        "dev": "tsc && concurrently 'vite build --watch' 'contentlayer dev'",
+        "build": "tsc && pnpm build:lib && contentlayer build ",
+        "build:lib": "vite build",
+        "preview": "vite preview"
+    },
+    "sideEffects": [
+        "**/*.css"
+    ],
+    "devDependencies": {
```

Comment:
`dependencies` before `devDependencies` (if the linter is correctly configured, it should complains already)